tests/spdx/parser/tagvalue/test_package_parser.py

```diff
@@ -83,6 +83,26 @@ def test_parse_package():
     assert package.valid_until_date == datetime(2022, 1, 1, 12)
 
 
+def test_parse_package_with_no_assertion_as_source_info():
```

Comment:
We might want to rename this to `test_parse_none_and_no_assertion_as_text` or something. Otherwise we emphasize the source info despite it not being a special case.

Reply:
I agree, I also moved the test to the `test_tag_valu_parser.py`module as it is not specific to packages.